docs/hosting/architecture/database-structure.md

```diff
@@ -19,20 +19,36 @@ To inspect the n8n database, you can use [DBeaver](https://dbeaver.io){:target=_
 
 These are the tables n8n creates during setup.
 
+### auth_identity
+
+Stores details of external authentication providers when using [SAML](/user-management/saml/). [TODO - confirm]
+
+### auth_provider_sync_history
+
+Stores the history of a SAML connection. [TODO - confirm]
+
 ### credentials_entity
 
 Stores the credentials used to authenticate with integrations.
 
-### eventdestinations_entity
+### event_destinations
 
 Enterprise only.
 
 Contains the destination configurations for [Log streaming](/log-streaming/).
 
+### execution_data
+
+Links executions to workflow data. [TODO - confirm]
```

Comment:
This is the actual execution data and contains the workflow at the time of running and the execution data